src/System.Device.Gpio/Interop/Unix/Libc/Interop.lseek.cs

```diff
@@ -6,7 +6,7 @@
 
 internal partial class Interop
 {
-    [DllImport(LibcLibrary, SetLastError = true)]
```

Comment:
Seems we are using LastError only for Interop open() mmap() methods, shouldn't we remove it from all other remaining methods? like read(), write() methods ...

Reply:
I did go and made sure that all the remaining SetLastError PInvokes actually call Marshal.GetLastWin32Error after it, which means that the ones that still have it, is because they need it.